docs/admin/installation.md

```diff
@@ -0,0 +1,55 @@
+# Installing DTS Locally
+
+Here we describe how to build, test, and install the Data Transfer Service
+in a local environment.
+
+## Building and Testing
+
+DTS is written in [Go](https://go.dev/), so you'll need a working Go compiler
+to build, test, and run it locally. If you have a Go compiler, you can clone
+this repository and build it from the top-level directory:
+
+```
+go build
+```
+
+### Running Unit Tests
+
+DTS comes with several unit tests that demonstrate its capabilities, and you can
+run these tests as you would any other Go project:
+
+```
+go test ./...
+```
+
+You can add a `-v` flag to see output from the tests.
+
+Because DTS is primarily an orchestrator of network resources, its unit tests
+must be able to connect to and utilize these resources. Accordingly, you must
+set the following environment variables to make sure DTS can do what it needs
+to do:
+
+* `DTS_KBASE_DEV_TOKEN`: a KBase development token (available to
```

Comment:
Which of the kbase environments does the token need to be for? There's prod / CI / appdev / various other ones and the auth server URL and token will differ depending on which env it is.

Reply:
The environment name is in the URL except in the case of prod, so it's `https://appdev.kbase.us/services/blahblahblah` for app dev, `https://ci.kbase.us/services/blahblahblah` for CI, and `https://kbase.us/services/blahblahblah` for prod. Maybe this is a KBase-specific detail that can be just skated over -- I don't know that there would be much call for a DTS instance connected to CI, and appdev and prod share the same data.

Reply:
We are indeed [using the production KBase auth server](https://github.com/kbase/dts/blob/jeff-cohere/demo/auth/kbase_auth_server.go#L33), so how about I just clarify this in the documentation?

Reply:
Yep, that sounds good.